template/lib.typ

```diff
@@ -89,6 +89,53 @@
   v(15pt)
 }
 
+#let court_info(
+  metadata
+) = {
+  pagebreak(weak: true)
+
+  set text(font: "TeX Gyre Heros", size: 20pt, stretch: 85%)
+  set block(spacing: 0pt)
+  
+  grid.with(columns: (90pt, auto), gutter: 10pt)(
+    text(size: 10pt, [Trabajo Fin de Grado: ]),
+    text(size: 10pt, metadata.title)
+  )
+
+  v(1cm)
+  text(size: 10pt, [Autor: ])
+  text(size: 10pt, metadata.author)
+  v(-0.65cm)
+  text(size: 10pt, [Tutor: ])
+  text(size: 10pt, metadata.tutor)
+  v(.5cm) 
+
+  text(size: 10pt, [El tribunal nombrado para juzgar el trabajo arriba indicado, compuesto por los siguientes profesores: ])
+  v(1.25cm)
+
+  grid.with(columns: (70pt, auto, 90pt), gutter: 0pt)(
+    v(1.5cm),
+    text(size: 10pt, [Presidente:]),
+    v(1.5cm),
+    v(3cm),
+    text(size: 10pt, [Vocal/es:]),
+    v(3cm),
+    v(1.5cm),
+    text(size: 10pt, [Secretario: ]),
+    v(1.5cm)
+  )
+
+  text(size: 10pt, [Acuerdan otorgarle la calificación de: ])
+  v(4.5cm)
+  grid.with(columns: (7.5cm,7cm), gutter: 10pt)(
+    v(3.5cm),
+    text(size: 10pt, [El Secretario del Tribunal:]),
+    v(1.5cm),
+    text(size: 10pt, [Fecha:]),
+    
+  )
+}
```

Comment:
Good job on this, however I don't understand why you set the font size to 20pt to later override it to 10pt. Why not set it to 10pt on the `#set` statement?

Reply:
Oh yes, I just correct that. It was a sily mistake sorry.